src/main/resources/lib/nextjs-proxy/proxy.es6

```diff
@@ -185,6 +202,23 @@ const proxy = function (req) {
     }
 };
 
+function getNextjsCookies() {
+    return COOKIE_CACHE.get(COOKIE_KEY, () => undefined);
+}
+
+function setNextjsCookies(cookies) {
+    removeNextjsCookies();
+    return COOKIE_CACHE.get(COOKIE_KEY, () => cookies);
+}
+
+function initNextjsCookieName() {
+    COOKIE_KEY = `NEXTJS_COOKIE_FOR_${portalLib.getSite()._name}`;
```

Comment:
Will this call getSite() for all pages? Or all pages that require the header data?
Possible to store this some way?

Reply:
Yes, this is called for every request because we decided to make this cookie unique per site.
So even if we cache it we'll have to check if the site is still the same by loading site.